frontend/components/SetupPage/SetupYourAgent/OptimusAgentForm/OptimusAgentForm.tsx

```diff
@@ -19,49 +20,68 @@ import {
 } from '../../../AgentForms/common/formUtils';
 import { InvalidGeminiApiCredentials } from '../../../AgentForms/common/InvalidGeminiApiCredentials';
 import {
+  CoinGeckoApiKeyDesc,
   CoinGeckoApiKeyLabel,
+  CoinGeckoApiKeySubHeader,
+  GeminiApiKeyDesc,
   GeminiApiKeyLabel,
+  GeminiApiKeySubHeader,
+  TenderlyAccessTokenDesc,
   TenderlyAccessTokenLabel,
   TenderlyAccountSlugLabel,
+  TenderlyApiKeySubHeader,
   TenderlyProjectSlugLabel,
 } from '../../../AgentForms/common/labels';
+import { RenderForm } from '../useDisplayAgentForm';
 import {
   OptimusFieldValues,
   useOptimusFormValidate,
 } from './useOptimusFormValidate';
 
-const { Text } = Typography;
-
-const SetupHeader = () => (
-  <Text>
-    Set up your agent with access to a{' '}
-    <a target="_blank" href={TENDERLY_URL}>
-      Tenderly
-    </a>{' '}
-    project for simulating bridge and swap routes, and swap routes and provide a{' '}
-    <a target="_blank" href={COINGECKO_URL}>
-      CoinGecko API key
-    </a>{' '}
-    as a price source.
-  </Text>
-);
-
-type OptimusAgentFormProps = { serviceTemplate: ServiceTemplate };
+type OptimusAgentFormContentProps = {
+  serviceTemplate: ServiceTemplate;
+  currentStep: BabyDegenFormStep;
+  updateNextStep: () => void;
+};
 
-export const OptimusAgentForm = ({
+const OptimusAgentFormContent = ({
```

Comment:
any chance we can share the same for for both modius and optimus? or it's too hard to handle